src/QuadForm/Torsion.jl

```diff
@@ -1698,3 +1698,33 @@ function _is_genus_brown(T::TorQuadMod, signature_pair::Tuple{Int, Int})
   return true
 end
 
+###############################################################################
+#
+#  Orthogonal sum
+#
+###############################################################################
+
+@doc Markdown.doc"""
+    orthogonal_sum(T::TorQuadMod, U::TorQuadMod)
+                                    -> TorQuadMod, TorQuadModMor, TorQuadModMor
+
+Return the orthogonal direct sum `S` of `T` and `U` as a quotient of the direct
+sum of their respective covers.
+
+It is given with the two injections $T \to S$ and $U \to S$.
+"""
+function orthogonal_sum(T::TorQuadMod, U::TorQuadMod)
+  cT = cover(T)
+  rT = relations(T)
+  cU = cover(U)
+  rU = relations(U)
+  cS, cTincS, cUincS = orthogonal_sum(cT, cU)
+  rS, _, _ = orthogonal_sum(rT, rU)
```

Comment:
This relies on caching of the ambient space and is better avoided.
```suggestion
  rS = lattice(ambient_space(cS), block_diagonal_matrix([basis_matrix(rU), basis_matrix(rV)]))
```